scripts/generateAssetData/generateRelatedAssetIndex/generateRelatedAssetIndex.ts

```diff
@@ -140,15 +196,20 @@ const processRelatedAssetIds = async (
   }
 
   const relatedAssetsResult = await getRelatedAssetIds(assetId, assetData)
+  const manualRelatedAssetsResult = getManualRelatedAssetIds(assetId)
 
   // ensure empty results get added so we can use this index to generate distinct asset list
-  const { relatedAssetIds, relatedAssetKey } = relatedAssetsResult ?? {
-    relatedAssetIds: [],
-    relatedAssetKey: assetId,
-  }
+  const { relatedAssetIds, relatedAssetKey } = manualRelatedAssetsResult ??
+    relatedAssetsResult ?? {
+      relatedAssetIds: [],
+      relatedAssetKey: assetId,
+    }
+
+  // Has zerion-provided related assets, or manually added ones
+  const hasRelatedAssets = Boolean(relatedAssetIds.length > 0)
```

Comment:
`Boolean()` a bit redundant :P honestly don't care, just wanted to poke fun haha

Reply:
Why all the hate for `Boolean` casts :(

Reply:
Lol, just saying `relatedAssetIds.length > 0` _is_ a boolean result already, can use `Boolean(relatedAssetIds.length)` and would be happy with that

Reply:
c6de76c9